controllers/users-controller.js

```diff
@@ -2,6 +2,8 @@ const User = require('../models/users');
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
 const { JWT_KEY } = require('../config');
+const validEmail;
```

Comment:
Close - we still want the validEmail and validPassword to be within the scope of registerUser function; not global functions. You just need to move the entire function codes before the if statement codes curently at line 27. Basically move lines 38 - 46 before if statement code logic. Apologies for the confusion.